tests/integration_tests/functional/test_vsock.py

```diff
@@ -225,54 +225,64 @@ def test_vsock_transport_reset_g2h(uvm_nano, microvm_factory):
     test_vm.api.vsock.put(vsock_id="vsock0", guest_cid=3, uds_path=f"/{VSOCK_UDS_PATH}")
     test_vm.start()
 
-    host_socket_path = os.path.join(
-        test_vm.path, f"{VSOCK_UDS_PATH}_{ECHO_SERVER_PORT}"
-    )
-    host_socat_commmand = [
-        "socat",
-        "-dddd",
-        f"UNIX-LISTEN:{host_socket_path},fork",
-        "STDOUT",
-    ]
-    host_socat = subprocess.Popen(
-        host_socat_commmand, stdout=subprocess.PIPE, stderr=subprocess.PIPE
-    )
-
-    # Give some time for host socat to create socket
-    time.sleep(0.5)
-    assert Path(host_socket_path).exists()
-    test_vm.create_jailed_resource(host_socket_path)
-
-    # Create a socat process in the guest which will connect to the host socat
-    guest_socat_commmand = f"tmux new -d 'socat - vsock-connect:2:{ECHO_SERVER_PORT}'"
-    test_vm.ssh.run(guest_socat_commmand)
-
-    # socat should be running in the guest now
-    code, _, _ = test_vm.ssh.run("pidof socat")
-    assert code == 0
-
     # Create snapshot.
     snapshot = test_vm.snapshot_full()
     test_vm.resume()
```

Comment:
why do we need to resume it here, although it'll be killed soon?

Reply:
Actually I need to resume it to check that after a snapshot socat is killed.